Add variable in app.js to setup the ajax timeout upload file

atomindtechnology opened the pull request:
In this way, we can control the ajax timeout on client side when we upload attachments

Commit:
Add variable in app.js to setup the ajax timeout upload file
In this way, we can control the ajax timeout on client side when we upload attachments

alecpl commented on program/js/app.js:
```diff
@@ -60,6 +60,7 @@ function rcube_webmail()
   this.identifier_expr = /[^0-9a-z_-]/gi;
 
   // environment defaults
+  this.uploadTimeout = 0 * 1000; // 0 = no timeout | ajax call timeout for loading attachment
```
Please, move it up to the "webmail client settings" section, and remove ` * 1000`.

Commit:
Added variable in app.js to setup the ajax timeout upload file
In this way, we can control the ajax timeout on client side when we upload attachments
Issue (https://github.com/roundcube/roundcubemail/issues/7548)
